modules/plugins/terminal/toggleterm/config.nix

```diff
@@ -50,9 +42,9 @@ in {
             end
           })
 
-          vim.keymap.set('n', ${toJSON cfg.lazygit.mappings.open}, function() lazygit:toggle() end, {silent = true, noremap = true, desc = 'Open lazygit [toggleterm]'})
+          vim.keymap.set('n', ${toJSON cfg.lazygit.mappings.open}, function() lazygit:toggle() end, {silent = true, noremap = true, desc = '${lazygitMapDesc}'})
```

Comment:
Might as well embed the mapping string directly as well, instead of relying on toJSON (I'm not sure why we do that anywhere in the first place, when toLuaObject exists).